crates/bevy_render/src/texture/image.rs

```diff
@@ -639,6 +657,7 @@ impl Image {
         #[allow(unused_variables)] supported_compressed_formats: CompressedImageFormats,
         is_srgb: bool,
         image_sampler: ImageSampler,
+        cpu_persistent_access: bool,
```

Comment:
I don't love the fact that there are now two bools; it makes code hard to read. Could we factor these out into an `ImageFlags` that contains `ImageFlags::SRGB` and `ImageFlags::CPU_PERSISTENT_ACCESS`?

Reply:
It's now an enum